### PR TITLE
Added method get wfs by correlation Ids and Names for WorkflowClient

### PR DESCRIPTION
**Pull request by gardusig**

### CODE REVIEW

**Review by c4lm — APPROVED** [review]

File: src/main/java/io/orkes/conductor/client/WorkflowClient.java [hunk 1]
```diff
@@ -68,6 +68,19 @@ public abstract BulkResponse terminateWorkflowsWithFailure(List<String> workflow
     public abstract Map<String, List<Workflow>> getWorkflowsByNamesAndCorrelationIds(
             List<String> correlationIds, List<String> workflowNames, Boolean includeClosed, Boolean includeTasks);
 
+    /**
```
Comment (v1r3n) [inline src/main/java/io/orkes/conductor/client/WorkflowClient.java:71]: this is going to be super confusing with methods returning EXACT same return value.

Comment (c4lm) [reply src/main/java/io/orkes/conductor/client/WorkflowClient.java:71]: not if we mark the old one as `@Deprecated`

Comment (coderabhigupta) [reply src/main/java/io/orkes/conductor/client/WorkflowClient.java:72]: @v1r3n How do you feel about moving ahead with this, given that we're deprecating the above method?

Comment (v1r3n) [reply src/main/java/io/orkes/conductor/client/WorkflowClient.java:72]: let's do it.

**Review by v1r3n — APPROVED** [review]

**Pull request merged**